workflow-templates/test-package.yaml

```diff
@@ -0,0 +1,76 @@
+name: Test
+
+on:
+  # Run CI on pushes to the main branch, and on PRs against main.
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+env:
+  PUB_ENVIRONMENT: bot.github
+
+jobs:
+  # Check code formatting and static analysis on a single OS (linux)
+  # against Dart stable and dev.
+  analyze:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        sdk: [dev, stable]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache Pub hosted dependencies
+        uses: actions/cache@v2
+        with:
+          path: "~/.pub-cache/hosted"
+          key: "os:ubuntu-latest;pub-cache-hosted;dart:${{ matrix.sdk }};packages:${{ github.repository }}"
+          restore-keys: |
+            os:ubuntu-latest;pub-cache-hosted;dart:${{ matrix.sdk }};packages:${{ github.repository }}
+            os:ubuntu-latest;pub-cache-hosted;dart:${{ matrix.sdk }}
+            os:ubuntu-latest;pub-cache-hosted
+            os:ubuntu-latest
+      - uses: dart-lang/setup-dart@v0.1
+        with:
+          channel: ${{ matrix.sdk }}
+      - name: Report version
```

Comment:
This should no longer be needed as we added it to setup-dart

Reply:
Done.